packages/fuels-accounts/src/lib.rs

```diff
@@ -396,7 +398,7 @@ mod tests {
         let mut tx: ScriptTransaction = FuelTransaction::script(
```

Comment:
Since we are referring to this code in the docs, we should change it to use our tx builders and input types.
Just leaving this as a reminder, no need to change it in this PR

Reply:
Can you please make an issue for this ?

Reply:
#982